lib/thrift/exceptions.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Thrift.Exceptions do
+  defmodule TApplicationException do
+    defexception message: nil, type: nil
+  end
+end
```

Comment:
I'd vote for t_application_exception.ex probably. Doesn't seem like a big deal though. It's the only built-in exception right?